cmd/kubeless/function_test.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"k8s.io/client-go/pkg/api/v1"
+	"reflect"
+	"testing"
+)
+
+func TestParseLabel(t *testing.T) {
+	labels := []string{
+		"foo=bar",
+		"bar:foo",
+		"foobar",
```

Comment:
:+1:   Add a `foo=bar=baz` case too, just to make sure we're getting the left-most.